src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -263,6 +269,8 @@ private void configureBindings() {
                 .deadlineWith(Commands.idle(shooter)))
         .toggleOnTrue(led.raindrop());
 
+    operator.y().onTrue(shooting.feedToAmp(x, y));
```

Comment:
Maybe this should be `.whileTrue`? Currently in simulation, it constantly tracks the feeding target area, even after the note is shot there.